### PR TITLE
New package: linux5.7

### PR DESCRIPTION
**Pull request by foxlet**
There's new modules to support exFAT, iOS fast charging, and more, so a review for anything else that should be enabled would be appreciated.

### CODE REVIEW

File: srcpkgs/linux5.7/template [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+	for file in $(find arch/${subarch:-$arch} -name module.lds -o -name Kbuild.platforms -o -name Platform); do
+		mkdir -p ${hdrdest}/$(dirname $file)
+		install -Dm644 $file ${hdrdest}/${file}
+	done
+	mkdir -p ${hdrdest}/include
+	# Remove firmware stuff provided by the "linux-firmware" pkg.
+	rm -rf ${DESTDIR}/usr/lib/firmware
+
+	for i in acpi asm-generic clocksource config crypto drm generated linux \
+		math-emu media net pcmcia scsi sound trace uapi video xen dt-bindings; do
```
Comment (daniel-eys) [inline srcpkgs/linux5.7/template:183]: Please add `vdso` here, several `linux/*.h` now `#include` these.

**Comment by renatoaguiar** [comment]
It fails to build for aarch64-musl:

```
In file included from ./include/linux/wireless.h:13,
                 from drivers/net/wireless/ath/wil6210/wil6210.h:12,
                 from drivers/net/wireless/ath/wil6210/main.c:12:
./include/uapi/linux/wireless.h:77:10: fatal error: stddef.h: No such file or directory
   77 | #include <stddef.h>                     /* for offsetof */
      |          ^~~~~~~~~~


In file included from ./include/linux/wireless.h:13,
                 from ./include/net/iw_handler.h:201,
                 from net/core/dev.c:112:
./include/uapi/linux/wireless.h:77:10: fatal error: stddef.h: No such file or directory
   77 | #include <stddef.h>                     /* for offsetof */
      |          ^~~~~~~~~~
```

**Comment by ericonr** [comment]
To track issues https://bbs.archlinux.org/viewtopic.php?id=256507

**Comment by foxlet** [comment]
I am using a Ryzen-based system with 5.7.2 and had no problems with KDE Plasma, however it does seem like there are a couple of issues to patch with this release.

**Pull request closed without merging**